src/generators/operationGenerator.ts

```diff
@@ -909,12 +1009,20 @@ function writeMultiMediaTypeOperationBody(
   overloadParameterDeclarations: ParameterWithDescription[][],
   responseName: string,
   clientDetails: ClientDetails,
-  isInline = false
+  isInline = false,
+  optionsBag: OptionsBag
 ): void {
-  operationMethod.addStatements([
-    "let operationSpec: coreHttp.OperationSpec;",
-    "let operationArguments: coreHttp.OperationArguments;"
-  ]);
+  if (!optionsBag.useCoreV2) {
```

Comment:
nit: you could also do something like this

```typescript
const coreImport = !optionsBag.useCoreV2 ? "coreHttp" :  "coreClient"
operationMethod.addStatements([      
  `let operationSpec: ${coreImport}.OperationSpec;`,
  `let operationArguments: ${coreImport}.OperationArguments;`
 ]);


```

Reply:
Done